### PR TITLE
Code style suggestions

### LINKED ISSUE #164: Replace "NO DATA" with null

### PR DESCRIPTION
**Pull request by romfrolov**
- No `var`
- Added missing semicolons
- Uppercase letters for base URL constant

---
Closes #164.
Closes #166.

### CODE REVIEW

File: funcs/historical.js [hunk 1]
```diff
@@ -36,6 +35,7 @@ var historical = async (keys, redis) => {
   // to store parsed data
   const result = [];
   const timelineKey = parsedCases[0].splice(4);
+  // REVIEW
   // parsedCases.pop()
   // parsedDeaths.pop()
   // recParsed.pop()
```
Comment (romfrolov) [inline funcs/historical.js]:
```suggestion
```
We should either remove these lines or add some meaningful comment about why these lines should remain commented out.

File: utils/country_utils.js [hunk 2]
```diff
@@ -560,11 +560,11 @@ const getCountryData = function (countryName) {
             };
         }
     }
-    return { iso2: 'NO DATA', iso3: 'NO DATA', _id: 'NO DATA', lat: 0, long: 0, flag: 'https://raw.githubusercontent.com/NovelCOVID/API/master/assets/flags/unknow.png' };
+    return { iso2: null, iso3: null, _id: null, lat: 0, long: 0, flag: 'https://raw.githubusercontent.com/NovelCOVID/API/master/assets/flags/unknow.png' };
```
Comment (ebwinters) [inline utils/country_utils.js]: #117 this PR will have merge conflicts ...

**Comment by romfrolov** [comment]
@ebwinters I returned the previous values to avoid conflicts and keep PRs independent.

**Comment by EliteDaMyth** [comment]
not required and is conflicting with some changes ;(

**Pull request closed without merging**